docker-entrypoint-development-sidekiq.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+set -e
+
+echo "Checking gem dependencies"
+bundle check \
+  || (
+    echo "Installing missing gems" \
+      && bundle install --jobs $(expr $(nproc) - 1) --retry 3
+  )
+
+echo "Run migrations or create DBs with seeds"
+(rails db:exists && rails db:migrate) || rails db:setup db:test:prepare
+
+mkdir -p tmp/cache tmp/pids tmp/sessions tmp/sockets
+
+exec "$@"
```

Comment:
These changes should be in entrypoint which is executed by each service which uses the app image like web_service, sidekiq_service and any other if any like pollers etc.